rj_msgs/CMakeLists.txt

```diff
@@ -22,12 +22,14 @@ rosidl_generate_interfaces(
   # Messages
   msg/BallState.msg
   msg/BallPlacement.msg
+  msg/Coach.msg
```

Comment:
let's name this something more descriptive. "PlayCall" seems semantically close to what the Coach node is actually doing. @anshulm07 any ideas?

Reply:
actually, "PlayCall" should be reserved for the allocation of defenders/attackers that the coach node will output. (this is also urgent for the next PR @p-nayak11 ). maybe "CoachMatchState" on topic strategy/coach_match_state?